static/src/javascripts.flow.archive/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -363,7 +318,7 @@ const sonobiBidder: PrebidBidder = {
                 appNexusTargeting: buildAppNexusTargeting(getPageTargeting()),
                 pageViewId: config.get('ophan.pageViewId'),
             },
-            isInSafeframeTestVariant() ? { render: 'safeframe' } : {}
+            {}
```

Comment:
I think this can be safely removed, along with the whole `Object.assign()`, as there’s now only a single object. See [MDN docs][1]

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign